### PR TITLE
fixing alignments

### PR DESCRIPTION
**Pull request by lubacien**
Hello,

This was not working as it created a list of segments which was appended instead of appending each segment separately. Now it seems to work.

thanks

### CODE REVIEW

File: lhotse/supervision.py [hunk 1]
```diff
@@ -671,7 +671,8 @@ def with_alignment_from_ctm(
                     num_overspanned += len(alignment)
                     segments.append(fastcopy(seg, alignment={type: alignment}))
             else:
-                segments.append([s for s in self.find(recording_id=reco_id)])
+                for s in self.find(recording_id=reco_id):
```
[the remark this answers is not in the document]
Comment (pzelasko) [reply lhotse/supervision.py:674]: It should be possible to do even `segments.extend(self.find(recording_id=reco_id))`. 

Can we add a unit test to cover this case?

**Comment by pzelasko** [comment]
Thanks for your first contribution! Can you also help us with testing the code?

**Comment by lubacien** [comment]
ok, I will work on it

**Comment by lubacien** [comment]
I found another error:
`    def write_alignment_to_ctm(self, ctm_file: Pathlike, type: str = "word") -> None:
        """
        Write alignments to CTM file.
    
        :param ctm_file: Path to output CTM file (will be created if not exists)
        :param type: Alignment type to write (default = `word`)
        """
        with open(ctm_file, "w") as f:
            for s in self:
>               if type in s.alignment:
E               TypeError: argument of type 'NoneType' is not iterable`

When there is no recording_id found, we do not write any alignment, but when there is a recording id without an alignment we write {type: []}. 
Now this is also solved, by adding an empty alignment also if we do not find the recording_id.

**Review by desh2608 — APPROVED** [review]

**Pull request merged**